pkg/kubectl/describe.go

```diff
@@ -94,12 +98,16 @@ func DescribableResources() []string {
 
 // Describer returns the default describe functions for each of the standard
 // Kubernetes types.
-func DescriberFor(kind string, c *client.Client) (Describer, bool) {
-	f, ok := describerMap(c)[kind]
-	if ok {
-		return f, true
+func DescriberFor(kind string, c client.VersionInterface) (Describer, bool) {
```

Comment:
I understand you use VersionInterface because it's a common interface implemented by both Client and ExpClient, but that's not the supposed use of VersionInterface. Can we define an interface similar to IsAnAPIObject?